server/src/test/java/org/elasticsearch/index/mapper/BinaryFieldMapperTests.java

```diff
@@ -121,9 +123,12 @@ public void testStoredValue() throws IOException {
 
     @Override
     protected Object generateRandomInputValue(MappedFieldType ft) {
-        assumeFalse("We can't parse the binary doc values we send", true);
-        // AwaitsFix https://github.com/elastic/elasticsearch/issues/70244
-        return null;
+        if (rarely()) {
+            return null;
+        } else {
```

Comment:
I think it'd be a little easier to read without the `else` bit. It'd not important, but my eyes are just used to the "early return" pattern.

Reply:
@nik9000 You are super fast at reviewing :)

Reply:
Addressed in 3aa1286d84fe1628740d286e77e7314a92dbe9e4